ironfish/src/wallet/wallet.ts

```diff
@@ -676,20 +631,11 @@ export class Wallet {
 
     for await (const decryptedNote of account.getUnspentNotes(assetIdentifier)) {
       if (minimumBlockConfirmations > 0) {
-        const transaction = await account.getTransaction(decryptedNote.transactionHash)
-
-        Assert.isNotUndefined(
-          transaction,
-          `Transaction '${decryptedNote.transactionHash.toString(
-            'hex',
-          )}' missing for account '${account.id}'`,
-        )
-
-        if (!transaction.sequence) {
+        if (!decryptedNote.sequence) {
```

Comment:
I know this was done in another PR but curious why we moved blockHash/sequence to the note since it seems like a denormalization if the note also has the transactionHash

Reply:
We added blockHash and sequence to the note for performance -- avoiding looking up the transaction to get the sequence for a note